Dfe.Data.SearchPrototype/Web/Tests/Unit/Models/ViewModels/Shared/ScrollablePagerTests.cs

```diff
@@ -0,0 +1,198 @@
+﻿using Dfe.Data.SearchPrototype.Web.Models.ViewModels.Shared;
+using FluentAssertions;
+using Xunit;
+
+namespace Dfe.Data.SearchPrototype.Web.Tests.Unit.Models.ViewModels.Shared
+{
+    public sealed class ScrollablePagerTests
```

Comment:
I haven't checked any of these tests because I need a verbal explanation of what's going on

Also, are they duplicates of other tests written for the PaginationResultsToPaginationViewModelMapper - I suggested they be moved to PartialIntegration but maybe the PaginationResultsToPaginationViewModelMapperTests just need to be made into unit tests

Reply:
i can do it but @spanersoraferty is the **boundary** and **threshold** master

Reply:
I've ensured all the tests that were in the mapper that were testing methods from this class are now moved here